### PR TITLE
PDM updated from 2.0.7 to 2.1.2

### PR DESCRIPTION
**Pull request by gretarg**

### CODE REVIEW

File: vendors/pdm-proto/include/protobuf.h [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
-namespace platform = eve_public::app::platform;
-#else
 #include "eve_launcher/pdm.pb.h"
-#endif
 
 namespace pdm_proto
 {
-    platform::Information GetData();
+	eve_public::app::platform::Information GetEVEPublicData();
+	platform::Information GetEVELauncherData();
```
Comment (rosabj) [inline vendors/pdm-proto/include/protobuf.h:9]: So, platform was eve_public::app::platform, but isn't anymore? I feel like I'm missing something.

File: src/libs/installer/installer.pro [hunk 2]
```diff
@@ -14,7 +14,8 @@ include(../../../installerfw.pri)
 #   SOURCES += $$PWD/productkeycheck.cpp
 #   ...
 #   your files if needed
-HEADERS += productkeycheck.h
+HEADERS += productkeycheck.h \
+    eve_public/app/platform.pb.h
```
Comment (hjaltisan) [inline src/libs/installer/installer.pro]: Why not declare this in line 52 instead (same place as the eve_launcher headers)?

File: src/libs/installer/installer.pro [hunk 3]
```diff
@@ -145,6 +146,7 @@ HEADERS += eventlogger.h \
     componentselectionpage_p.h
 
 SOURCES += eventlogger.cpp \
+    eve_public/app/platform.pb.cc \
```
Comment (hjaltisan) [inline src/libs/installer/installer.pro]: Same as above, why not have this in the same place as the eve_launcher pb.cc files (few lines down)?

File: vendors/info.txt [hunk 4]
```diff
@@ -3,9 +3,9 @@ sentry (0.4.9) (Qt IFW v1.15.0)
 Comes from release v4.0.0 of eve-launcher-vendor-libraries (https://github.com/ccpgames/eve-launcher-vendor-libraries/releases/tag/v4.0.0)
 Comes from TeamCity build tagged v4.0.0 (https://teamcity.evetech.net/buildConfiguration/Applications_EveLauncher_VendorLibraries_Packagers_SentryNativeMt?mode=builds)
 
-pdm-proto (2.0.7) (Qt IFW v1.15.1)
+pdm-proto (2.1.2) (Qt IFW v1.15.1)
```
Comment (hjaltisan) [inline vendors/info.txt]: The Qt IFW number in the parenthesis is to tell us in which version of our Qt IFW we updated to the currently used library.
Since we're making a change to the framework, we need to increase the Qt IFW number.
You do this by updating line 10 in `installerfw.pri`:
```
CCP_FRAMEWORK_STRING = 1.15.1
```
So you need to update that file, and then update the comment here to use the new number

File: vendors/pdm-proto/include/protobuf.h (hunk 1 above)
Comment (hjaltisan) [reply vendors/pdm-proto/include/protobuf.h:9]: I agree that this looks strange, but would be a question for Kristjan or Thomas since this is their code.

**Review by hjaltisan — CHANGES_REQUESTED** [review]
Main issue is that you're missing updating the Qt IFW version string in `installerfw.pri` (line 10)

File: vendors/pdm-proto/include/protobuf.h (hunk 1 above)
Comment (CCP-Aporia) [reply vendors/pdm-proto/include/protobuf.h:9]: The PDM now exposes the protobuf definitions from both, the `eve_launcher` and the `eve_public::app::platform` domains. 
Effectively, the old `GetData()` method was returning the same thing that `GetEVELauncherData()` is returning now, e.g. the protobuf definition sourced from the `eve_launcher` domain. `GetEVEPublichData()` on the other hand is new, and returning the protobuf definition from the `eve_public` domain.

In other words, you only need one or to the other, depending on which domain's definition you want (or need) to use.

CC @CCP-Lease

**Review by hjaltisan — APPROVED** [review]
Looks good
